CMakeLists.txt

```diff
@@ -133,6 +133,44 @@ endif()
 find_package(Threads REQUIRED)
 find_package(ZLIB REQUIRED)
 
+# try to find an installed boost
+set(boost_VERSION "1.71")
+find_package(Boost ${boost_VERSION} QUIET)
+if (NOT Boost_Found)
```

Comment:
few questions here:

- boost version to pin? this is ubuntu 20.04 currently. I have no real feelings about what we're pinning to, can be 1.77.0 if you want, I tried, works as well
- now I'm pretty sure the logic here is not robust. in src/CMakeLists.txt it seems we have to link `boost::headers` to the module targets (I couldn't compile without) and I guess that wouldnt be a "library" if it's not a conan installation.. maybe there's another way generally? more in the other cmakelists

Reply:
updated `boost::headers` to `boost::boost` in scr/CMakeLists.txt, seems to work as well. I'll try tmrw if it works with a system boost installation > 1.71. maybe there's a system out there which has boost >= 1.71 but not python3?  doesn't sound likely;) anyways, 160 MB of useless space saved is also an argument